samples/outlook/20-item-body/append-text-on-send.yaml

```
@@ -41,6 +41,10 @@ template:
 
         <section class="samples ms-font-m">
             <h3>Try it out</h3>
+            <p><b>Important</b>: To use <code>appendOnSendAsync</code>, you must set the <code>ExtendedPermission</code>
+                manifest element to <code>AppendOnSend</code>. To learn more about append-on-send and its configuration, see <a
```

If you accept this suggestion, do adjust as appropriate for display in the task pane
```suggestion
            <p><b>Important</b>: To use <code>appendOnSendAsync</code>, you must set <code>AppendOnSend</code> as an extended permission in the
             <code>ExtendedPermissions</code> manifest element. To learn more about append-on-send and its configuration, see <a
```